src/applications/personalization/profile/components/alerts/CredentialRetirementAlerts.jsx

```diff
@@ -0,0 +1,95 @@
+import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';
+import React from 'react';
+import { VaAlert } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { useSignInServiceProvider } from '../../hooks';
+import { HowToVerifyLink } from '~/platform/user/authorization/components/IdentityNotVerified';
+import { useSessionStorage } from '../../../common/hooks/useSessionStorage';
+
+// alerts to be used during the transition period of MHV and DS Logon credential retirement
+
+export const AccountSecurityLoa1CredAlert = () => {
+  const { label } = useSignInServiceProvider();
+
+  return (
+    <>
+      <VaAlert status="continue" visible uswds>
+        <h2 slot="headline">
+          Verify your identity with Login.gov or ID.me to manage your profile
+          information
+        </h2>
+        <div>
+          <p>
+            Before we give you access to your VA.gov profile, we need to make
+            sure you’re you—and not someone pretending to be you. This helps us
```

Comment:
Needs space around the hyphen. "you - and"

Reply:
Pulled this content directly from Figma so it does reflect what is in the design at this time. I will follow up with designer and see what they say. Thanks @Andrew565 for reviewing!